tests/tensorflow/nightly/vision-coco.libsonnet

```diff
@@ -49,6 +49,17 @@ local utils = import 'templates/utils.libsonnet';
       },
     },
   },
+  local maskrcnn = common.TfVisionTest + coco {
+    modelName: 'vision-maskrcnn',
+    scriptConfig+: {
+      experiment: 'maskrcnn_resnetfpn_coco',
+      paramsOverride+: {
+        task+: {
+          annotation_file: '$(COCO_DIR)/instances_val2017.json',
```

Comment:
Is `task.annotation_file` always required for coco? If so, this override should move to `coco`. (same for `retinanet`, even though it wasn't added in this PR)

Reply:
Yup looks like it, will do